product-service/serverless.ts

```diff
@@ -23,15 +23,60 @@ const serverlessConfiguration: Serverless = {
     apiGateway: {
       minimumCompressionSize: 1024,
     },
+    iamRoleStatements: [
+      {
+        Effect: "Allow",
+        Action: "sns:*",
+        Resource: { Ref: 'SNSTopic' }
+      }
+    ],
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
       PG_HOST: 'rss-app-database.cva9hwr0earm.eu-west-1.rds.amazonaws.com',
       PG_PORT: 5432,
       PG_DATABASE: 'postgres',
       PG_USERNAME: 'test',
-      PG_PASSWORD: 'test'
+      PG_PASSWORD: 'test',
+      SNS_ARN: { Ref: 'SNSTopic' }
     },
   },
+  resources: {
+    Resources: {
+      SQSQueue: {
+        Type: 'AWS::SQS::Queue',
+        Properties: {
+          QueueName: 'catalogItemsQueue'
+        }
+      },
+      SNSTopic: {
+        Type: 'AWS::SNS::Topic',
+        Properties: {
+          TopicName: 'createProductTopic'
+        }
+      },
+      SNSSubscription: {
+        Type: 'AWS::SNS::Subscription',
+        Properties: {
+          Endpoint: 'vkotov1@gmail.com',
+          Protocol: 'email',
+          TopicArn: { Ref: 'SNSTopic' },
+          FilterPolicy: {
+            products: 'send'
+          }
+        }
+      }
```

Comment:
```
set a Filter Policy for SNS createProductTopic in serverless.yml (Create an additional email subscription and distribute messages to different emails depending on the filter for any product attribute)
```
According to task, there should be an additional subscription, but you have only one

Reply:
Thanks, added, but used same email, cause don't want to create a new one)

Reply:
https://danq.me/2017/09/26/gmail-plus/
you can use this trick